worlds/animal-well/regions.py

```diff
@@ -350,8 +354,8 @@ class AWData(NamedTuple):
     bear_slink_room: {
         iname.slink:
```

Comment:
```suggestion
        lname.slink:
```

this is my bad but while we're here...